### PR TITLE
Adding Academy Mortgage domain

### PR DESCRIPTION
**Pull request by jswan106**
Would like to add the Academy Mortgage domain to bug-bounty-list.json

### CODE REVIEW

**Comment by jswan106** [comment]
@barnett Changed as requested, I think it should be good now.

**Comment by dantrauner** [comment]
FYI @jswan106 the `safe_harbor` field is an enum with `none` as an option – is that what you meant to add? You can also leave it blank for now if desired.

https://github.com/disclose/disclose/blob/master/bug-bounty-list/bug-bounty-list.schema.json#L21-L29

**Comment by jswan106** [comment]
@dantrauner Thanks for the info! I think we are going to leave it blank for now. :)

**Comment by jeffboothby** [comment]
@jswan106 Following up on this - checking again since I didn't see a change.

**Comment by jswan106** [comment]
@jeffboothby I believe that I have completed the requested changes, please take a look at the file in question:
image

Please let me know if you have any questions.

Thanks!

File: bug-bounty-list/bug-bounty-list.json [hunk 1]
```diff
@@ -129,6 +129,16 @@
     "hall_of_fame": false,
     "safe_harbor": "partial"
   },
+  {
+    "program_name": "Academy Mortgage",
+    "policy_url": "https://academymortgage.com/about-us/contact-us",
```
Comment (jeffboothby) [inline bug-bounty-list/bug-bounty-list.json]: @jswan106 https://academymortgage.com/about-us/contact-us doesn't list out the requirements for Partial Safe Harbor:

- Rewards: Whether compensation will be provided for (valid, unique) issues, as well as the form and magnitude of that compensation
- Official Communication Channels: An exhaustive list of the communication methods that are considered acceptable by the organization for receiving and communicating about any information associated with potential vulnerabilities
- Explicit permission to complete security research

Once these are met, we can add Academy Mortgage to the list, as well as marking Partial Safe Harbor.

**Comment by barnett** [comment]
@jswan106 going to close this as it's gone stale but feel free to re-open once it's been updated with @jeffboothby's comments https://github.com/disclose/disclose/pull/44#discussion_r294958942

**Pull request closed without merging**